### DISCUSSION TITLE
SQLitePCLRaw.bundle_green issue with sqlcipher if >= 2.0.2

### DISCUSSION THREAD

**Discussion by angelrishi** [ericsink/SQLitePCL.raw#510] [Q&A]
Hi,
I was using sqlite cipher of yours for encryption in the project as this:

<PackageReference Include="SQLitePCLRaw.bundle_green" Version="1.1.13" ExcludeAssets="All" />
<PackageReference Include="SQLitePCLRaw.bundle_sqlcipher" Version="1.1.13" />

But now Appcenter uses "SQLitePCLRaw.bundle_green" >= 2.0.2

So, I cannot use:

<PackageReference Include="SQLitePCLRaw.bundle_green" Version="2.0.2" ExcludeAssets="All" />
<PackageReference Include="SQLitePCLRaw.bundle_sqlcipher" Version="1.1.13" />

as it does not work. I even tried uninstalling "SQLitePCLRaw.bundle_sqlcipher" 1.1.13 and installing "SQLitePCLRaw.e_bundle_sqlcipher" 2.0.2. 
Still it does not work.

Please guide.

Thank you.
Angel

**Comment by angelrishi** [comment]
@ericsink , would be great if you can help. Thanks a lot.

**Comment by ericsink** [comment]
I probably need more info.  Saying "it does not work" isn't enough.

From what you've said, I can see a few things:

At some point, SQLCipher made a change that was not backwards compatible without an extra step.  That might be going on.

You also mention SQLitePCLRaw 1.x, which is not compatible with the 2.x series.  They can't be mixed.

I'll need to know what your target platform is, what nuget packages are involved, what error you are getting, etc.
